ask-sdk-local-debug/tst/com/amazon/ask/localdebug/client/WebSocketClientImplTest.java

```diff
@@ -46,6 +47,7 @@
 @PrepareForTest({
         RequestResponseUtils.class
 })
+@PowerMockIgnore({"javax.management.*", "javax.script.*"})
```

Comment:
What is this Ignore for ?

Also, I see you applied it across multiple packages
Maybe use a configuration file instead for eg: https://github.com/powermock/powermock-examples-maven/blob/master/global-ignore/src/test/resources/org/powermock/extensions/configuration.properties

Reply:
without those ignores there would be power mock errors reported, I added them to global config file as you suggested